rest-api-spec/src/main/resources/rest-api-spec/api/indices.put_template.json

```diff
@@ -33,6 +33,10 @@
         "flat_settings": {
            "type": "boolean",
            "description": "Return settings in flat format (default: false)"
+        },
+        "version": {
```

Comment:
Maybe I am missing something, but I don't think we support `version` as a URL parameter. We do support it in the body of the template, but not on URL. And now that I am thinking about it we don't support `flat_settings` here either. So, we should probably remove it.

Reply:
I think that we do not support `timeout` either. But we support `cause` which is not listed in the spec.  Also I am confused by `template` and `index_patterns` which are not specified in the spec. I guess we are missing some test coverage here as well, otherwise tests for these params would fail.